bigquery/api/BigqueryTest/BigqueryTest.cs

```diff
@@ -478,6 +455,35 @@ public void TestImportDataFromFile()
             Assert.Equal(uploadTestWordValue, row["unique_words"]);
         }
 
+        [Fact]
+        public void TestImportDataFromJSON()
+		{

```

Comment:
Weird things going on with indentations/spaces in this diff. Double check the indentation of all of the code, overall. This whole test method is indented too much.

Reply:
Done. I hadn't yet configured VS Community to convert tabs to spaces.